.github/eslint.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Copied from https://github.com/actions/setup-node/blob/master/.github/eslint-stylish.json . Do we need to add a copyright notice and if yes where? I would have added a comment in the file but json doesn't support comments.